fuzzing/generate_seed_eth_pkg.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python2
+
+import os
+import stat
+import subprocess
+
+def main():
+    with open("broken","rb") as f:
+        byte = bytearray(f.read())
+
+    # mov eax,04
```

Comment:
What is this file for ? If this is about ethernet headers, why is there assembly in the comments ?